Makefile

```
@@ -139,7 +138,10 @@ delete-integration-test-dev-cluster: build ## Delete the test cluster for use wh
 ##@ Code Generation
 
 .PHONY: generate-always
-generate-always: pkg/addons/default/assets/aws-node.yaml ## Generate code (required for every build)
+generate-always: ## Generate code (required for every build)
+	go install github.com/maxbrunsfeld/counterfeiter/v6
+	go install github.com/vektra/mockery/v2
+	go install github.com/vburenin/ifacemaker
```

super nit: we can move the go installs under a differ make rule and just call it here:
```suggestion
generate-always: install-gen-tools ## Generate code (required for every build)
```

---

good call - added a new target for installing tools https://github.com/eksctl-io/eksctl/pull/8125/commits/3331498639878cdaccec4cc26d4dba4c33120cbd

should this target run automatically where tools are required, or should that be an explicit step that CI/users take on their own